src/fsharp/CompileOps.fs

```diff
@@ -2702,8 +2702,6 @@ type TcConfig private (data : TcConfigBuilder,validate:bool) =
         | Some(fslibFilename) ->
             let filename = ComputeMakePathAbsolute data.implicitIncludeDir fslibFilename
             try 
-                use ilReader = OpenILBinary(filename,data.optimizeForMemory,data.openBinariesInMemory,None,None, data.shadowCopyReferences)
```

Comment:
We check this again later, see other calls to ``checkFSharpBinaryCompatWithMscorlib``.  there is no need to do it here - opening the binary fails in Visual Studio if the binary doesn't exist on disk